src/helper/utils/exception.js

```diff
@@ -0,0 +1,11 @@
+class ResponseException {
+  constructor(message) {
+    this.message = message;
+  }
+
+  toString() {
+    return (this.message);
```

Comment:
Should add toString() in return  to ensure function return String
```suggestion
    return (this.message.toString());
```

Reply:
That true. I will fix it.